microsoft/vscode-azureappservice
# Allow users to deploy zip files directly

Tested it with a zip file and it seems like it works perfectly fine.

How to test:
1. Clone a sample project (this one works fine https://github.com/johnpapa/node-hello)
2. Zip its contents (make sure that you don't zip the folder itself)
3. Right-click and deploy
